frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.html

```diff
@@ -32,6 +32,7 @@
     class="spot-list--item-floating-actions op-file-list--item-floating-actions hidden-for-mobile"
   >
     <button
+      type="button"
```

Comment:
Thanks! Yes, you're absolutely right every button should have an explicit type, but apparently it was forgotten here.

Reply:
In fact, it's already fixed on `dev`. This PR is to fix it for next 12.4.x.